#**Esse projeto simula uma demananda da empresa Airbnb**

As perguntas do CEO
1. Qual o valor médio do aluguel na cidade de Nova York?
2. Quais os nomes das regiões que existem na cidade de Nova York?
3. Qual o valor do aluguel diário mais caro da cidade de Nova York?

Depois de receber essa demanda do CEO, eu faço os processos abaixo com os dados recebidos.

# **Importações**

In [2]:
##**O ARQUIVO CSV É DE SUMA IMPORTÂNCIA, ALTERE O CAMINHO DO ARQUIVO DE ACORDO COM O AMBIENTE DE EXECUÇÃO**##

import pandas as pd
import numpy as np
from google.colab import drive

drive.mount('/content/drive')

#altere o caminho de acordo com seu ambiente de execução
arquivo_dados = '/content/drive/My Drive/python data science/AB_NYC_2019.csv'

#cria um DataFrame com o arquivo
data = pd.read_csv(arquivo_dados)

Mounted at /content/drive


# **Tasks**

In [3]:
# Seleciona e armazena a coluna que tem os preços
coluna = 'price'  # Define a coluna de interesse como 'price'


# Usa a função loc do pandas para selecionar todos os dados da coluna 'price'
preco_aluguel = data.loc[:, coluna]  # Armazena todos os valores da coluna 'price' na variável preco_aluguel


# Calcula a média dos preços de aluguel usando a função mean do numpy
preco_medio = np.mean(preco_aluguel)  # Calcula a média dos valores em preco_aluguel e armazena o resultado na variável preco_medio


# Seleciona e armazena a coluna 'neighbourhood_group'
regiao = data.loc[:, 'neighbourhood_group']  # Armazena todos os valores da coluna 'neighbourhood_group' na variável regiao


# Usa a função unique do pandas para obter os valores únicos na coluna 'neighbourhood_group'
nome_regiao = pd.unique(regiao)  # Armazena os valores únicos em regiao na variável nome_regiao


# Calcula o valor máximo dos preços de aluguel usando a função max do numpy
aluguel_caro = np.max(preco_aluguel)  # Armazena o valor máximo em preco_aluguel na variável aluguel_caro

## Printa o resultado da task

In [4]:
print(f"Com isso eu chego a conclusão de que o preço medio do aluguel é de {preco_medio:.2f}, \
\n Os nomes das regiões {nome_regiao} \
\n o valor do aluguel diário mais caro da cidade de Nova York é {aluguel_caro} ")

Com isso eu chego a conclusão de que o preço medio do aluguel é de 152.72, 
 Os nomes das regiões ['Brooklyn' 'Manhattan' 'Queens' 'Staten Island' 'Bronx'] 
 o valor do aluguel diário mais caro da cidade de Nova York é 10000 




---
---

---
---

---
---

# **E-mail com as respostas**

Olá CEO, espero que esse e-mail o encontre bem.
Meu nome é João victor, sou Cientista de Dados do time de expansão.
Segue abaixo as respostas das perguntas de negócio solicitadas, referente a
*expansão de Nova York*.


## Qual o valor médio do aluguel na cidade de Nova York?
R: O valor médio do aluguel é de U$ 152.72


## Quais os nomes das regiões que existem na cidade de Nova York?
R: Brooklyn, Manhattan, Queens, Staten Island, Bronx


## Qual o valor do aluguel diário mais caro da cidade de Nova York?
R: O valor do aluguel diária mais cara é de U$ 10.000,00

---
---
